### PR TITLE
feat: add watch mode only wallet address

### PR DESCRIPTION
**Pull request by drewcook**
## General Changes

- Adds new watch mode only feature for users to be able to view the protocol from the position of any wallet
- Stores a new `localStorage` variable for the watch mode wallet address

## Developer Notes

This PR stems from https://github.com/aave/interface/pull/1062 and adds code style changes that adhere to best practices.

## Author Checklist

- [x]  The base branch is set to `main`
- [x]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [x]  The Github issue has been linked to the PR in the Development section
- [x]  The General Changes section has been filled out
- [x]  Developer Notes have been added (optional)

## Reviewer Checklist

- [ ]  End-to-end tests are passing without any errors
- [x]  Code style generally follows existing patterns
- [x]  Code changes do not significantly increase the application bundle size
- [x]  New third-party packages, if any, do not introduce potential security threats
- [x]  There are no CI changes, or they have been OK’d by the devops team
- [ ]  Code changes have been quality checked in the ephemeral URL
- [x]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [x]  Squash and merge has been checked

### CODE REVIEW

**Comment by drewcook** [comment]
This style update should be made to fix this side-effect https://github.com/aave/interface/pull/1062#issuecomment-1259314941

File: src/components/WalletConnection/WalletSelector.tsx [hunk 1]
```diff
@@ -150,6 +161,57 @@ export const WalletSelector = () => {
...
+      </Box>
+      <Box
+        sx={(theme) => ({
+          p: '8px 12px',
+          border: `1px solid ${theme.palette.divider}`,
+          borderRadius: '6px',
+          mb: 1,
+        })}
+      >
+        <InputBase
```
Comment (grothem) [inline src/components/WalletConnection/WalletSelector.tsx:178]: One thing I noticed on mobile, iOS is zooming in on this input automatically. Once you enter the address and start watching, then you have to zoom back out. We could conditionally set the font size of the input to be `16px` if we're on small screens, then it won't zoom on focus. I just did something similar for the market search input.

**Comment by MareskoY** [comment]
@drewcook  found a UI glitch 

at the loading of the page at the first moment on wallet appear warning icon(as appearing on watch mode only), I think it's good to fix that for better quality 

warning-on-wallet

also from automation coverage, I found  new behavior for the wallet button is not clickable at the first second on the load page

**Comment by grothem** [comment]
@MareskoY I fixed the ui glitch with the watch icon showing at the start of cypress tests. In the setup for the tests, we were setting the `watchModeOnlyAddress` which I don't think makes sense for all cypress tests. I removed that step in the config setup.

**Review by defispartan — DISMISSED** [review]
👻🚀

**Review by defispartan — APPROVED** [review]
👻 🚀

**Review by grothem — APPROVED** [review]

**Pull request merged**